scripts/release.sh

```diff
@@ -0,0 +1,31 @@
+# /bin/bash
+
+set -e
+
+VERSION=${1}
+
+if [ -z "${VERSION}" ]; then
+    echo "usage: $0 <version>"
+    exit 1
+fi
+
+rm -rf build/
+mkdir build/
+cd build/
+
+# Copy the WASM binary
+cp ../reproducible-builds/wasm-client.wasm lnc-v${VERSION}-alpha.wasm
```

Comment:
Should we remove the `-alpha` and the `v` prefix/suffix and instead expect the version to be in the full format? Otherwise we'd probably forget to change this when we go from alpha to beta.